<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mWindowMain</class>
 <widget class="QMainWindow" name="mWindowMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="mButtonDrink1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>공체 Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1200원</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mButtonPurchase">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>740</y>
      <width>160</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>공체 Medium</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>결제하기</string>
    </property>
   </widget>
   <widget class="QLabel" name="mLabelDrinkName">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>740</y>
      <width>160</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>공체 Medium</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mLabelDrinkPrice">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>740</y>
      <width>280</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>공체 Medium</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mLabelDrinkName1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>공체 Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>코카콜라</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mLabelStock1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>245</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>공체 Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>현재 재고 : 20</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mLabelStock1_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>245</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>공체 Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>현재 재고 : 20</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mLabelDrinkName1_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>210</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>공체 Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>칠성사이다</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="mButtonDrink2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>공체 Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1100원</string>
    </property>
   </widget>
   <widget class="QLabel" name="mLabelStock1_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>585</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>공체 Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>현재 재고 : 20</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mLabelDrinkName1_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>550</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>공체 Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>환타 오렌지맛</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="mButtonDrink3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>620</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>공체 Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1000원</string>
    </property>
   </widget>
   <widget class="QLabel" name="mLabelStock1_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>585</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>공체 Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>현재 재고 : 20</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mLabelDrinkName1_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>550</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>공체 Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>마운틴듀</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="mButtonDrink4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>620</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>공체 Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>900원</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>40</y>
      <width>70</width>
      <height>160</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/Cocacola.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>40</y>
      <width>70</width>
      <height>160</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/Chilsung.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>380</y>
      <width>70</width>
      <height>160</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/Fanta.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>380</y>
      <width>70</width>
      <height>160</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/Mountaindew.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
